dependencies/buster/redfish_client/control

```diff
@@ -0,0 +1,27 @@
+Source: ruby-redfish-client
+Section: ruby
+Priority: optional
+Maintainer: Debian Ruby Extras Maintainers <pkg-ruby-extras-maintainers@lists.alioth.debian.org>
+Uploaders: Lukas Zapletal <lzap+pub@redhat.com>
+Build-Depends: debhelper (>= 11~),
+               gem2deb,
+               git,
+               ruby-excon (>= 0.60),
+               ruby-server-sent-events (>= 0.1)
+Standards-Version: 4.3.0
+Vcs-Git: https://salsa.debian.org/ruby-team/ruby-redfish-client.git
+Vcs-Browser: https://salsa.debian.org/ruby-team/ruby-redfish-client
```

Comment:
This is not really true, so please drop :)

Reply:
Sure.